.circleci/config.yml

```diff
@@ -270,15 +270,20 @@ jobs:
       DOCKER_REPOSITORY: unlockprotocol
     steps:
       - *cached-checkout
-      - run:
-          name: Set GIT_COMMIT_DESC
-          command: echo 'export GIT_COMMIT_DESC=$(git log --format=%B -n 1 "$CIRCLE_SHA1")' >> $BASH_ENV
       - run:
           name: Set IS_FORKED_PR
-          command: ./scripts/circleci/set-is-forked-pull-request.sh >> $BASH_ENV
+          command: scripts/circleci/set-is-forked-pull-request.sh >> $BASH_ENV
```

Comment:
any reason you changed this `./`? (I do not think this is  a problem  as long as `set-is-forked-pull-request.sh` is executable  but wanted to make sure youve seen this?)

Reply:
good catch, this is a cut and paste error